esp/src/src/Utility.ts

```diff
@@ -99,14 +99,22 @@ export function espTime2SecondsTests() {
 
 export function convertedSize(intsize: number): string {
     const unitConversion = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
-    if (intsize === null || intsize === undefined) {
+    if (intsize === null || intsize === undefined || intsize === -1) {
```

Comment:
Is the "-1" safe for all calls to this function?  It might be better to have an additional param option `treatMinusOneAsUndefined`?
Or have the -1 === undefined logic closer to where the data comes back from the server or in the grid formatter?